udaykalra/walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/style.css

```diff
@@ -7,3 +7,7 @@
 #quote-container {
   margin-top: 20px;
 }
+
+body {
+    background: white;
```

Comment:
Style prescribes a 2-space indent:
https://google.github.io/styleguide/htmlcssguide.html#Indentation

Does `make pretty` fix this?

Reply:
This is an example demo, not the webpage.